bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="broadlink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="sample">
+		<!-- Comment this in for Things which need a Bridge to function.
+			<supported-bridge-type-refs>
+			<bridge-type-ref id="BroadlinkBridge" />
+			</supported-bridge-type-refs>
+		-->
+
+		<label>Broadlink Binding Thing</label>
+		<description>Sample thing for Broadlink Binding</description>
+
+		<channels>
+			<channel id="channel1" typeId="sample-channel"/>
```

Comment:
Looks like this whole file needs to be deleted?

Reply:
Yup, this seems to be part of some initial boiler plate code that was never included in the binding in the end. Removed the associated files.